typespec-tests/src/main/java/com/type/property/additionalproperties/models/IsFloatAdditionalProperties.java

```diff
@@ -26,7 +26,7 @@ public final class IsFloatAdditionalProperties implements JsonSerializable<IsFlo
     private final double id;
 
     /*
-     * The model is from Record<float32> type.
+     * Dictionary of ExtendsUnknownAdditionalPropertiesDerivedAge
      */
```

Comment:
A bit confused, the dictionay is `Map<String, Double>`, why is this 'Dictionary of `ExtendsUnknownAdditionalPropertiesDerivedAge`', not dictionay of `Double`? Is it saying it's the `age` property of `ExtendsUnknownAdditionalPropertiesDerived` model?

Reply:
Yeah, this is a bit strange. (seems from the doc of the Record<>)

I will take a look.

Reply:
The cause is the `float` got names as `ExtendsUnknownAdditionalPropertiesDerivedAge`. This alone won't affect anything, as we don't generate a float class/struct.

But the `namer` module would find that there is no `@doc` on the Record, and use the name above to coin a description, hence the strange name.

Reply:
fix in https://github.com/Azure/autorest.java/pull/2499/commits/51f4ea5cd0c3869882c1c41f42217d39cd312dcc and https://github.com/Azure/autorest.java/pull/2499/commits/7f5c7434f7dc22cba074a521aeee00df2dc938ee

also disabled `namer` in test, and hide it in readme.